sockets/1.2/modules/ROOT/pages/sockets-documentation.adoc

```diff
@@ -594,7 +594,8 @@ Define your own custom protocol by writing a class that extends `TcpProtocol`.
 [[eof-protocol]]
 === Eof Protocol
 
-Reading is terminated by the stream being closed by the client.
+Reading is terminated by the client by shutting down its output stream, also known as TCP Half Close feature. Use this protocol with SSL sockets that use TLS 1.3 version and later, (Half Close feature was introduced in TLS 1.3).
```

Comment:
It would be better to start with what the Eof Protocol actually is and what it does. (I don't understand this well enough to reword.) Does the client shut down its own output stream or does the protocol do this?

Protocol that...